src/containers/AccountModal/forms/PersonalDetails.tsx

```diff
@@ -117,7 +119,10 @@ const PersonalDetails = () => {
             value: questionValues[question.key],
           } as CaptureCustomAnswer),
       );
-      await updateCaptureAnswers(removeEmpty({ ...formData, customAnswers }));
+
+      const metadata = removeEmpty({ ...user?.metadata });
+
+      await updateCaptureAnswers(removeEmpty({ ...formData, customAnswers, metadata }));
```

Comment:
Looks like there are hidden type problems here. The controller `updateCaptureAnswers` function only expects an arg of type `Capture`, which doesn't have a `metadata` property. Typescript only allows it because the `removeEmpty` function converts it to a Record type, but if you remove it temporarily you'll see the type error. `metadata` also doesn't exist on `UpdateCaptureStatusArgs` on the service, so it's actually only when it gets to the call to `updateCustomer` in the inplayer service that it is using the right type. This is too fragile of an approach. It can easily be broken with code cleanup and it will be quite hard to debug.